### PR TITLE
Palashg7563/husky fix

### PR DESCRIPTION
**Pull request by palashg7563**
this pr fixes the #67 and will the listing-fix before commiting

### CODE REVIEW

**Comment by palashg7563** [comment]
@abhishek71994 do review this one

File: server-api/package.json [hunk 1]
```diff
@@ -31,5 +32,10 @@
     "eslint-plugin-import": "2.17.2",
     "eslint-plugin-jsx-a11y": "6.2.1",
     "prettier-eslint-cli": "^4.7.1"
```
Comment (abhishek71994) [inline server-api/package.json]: Maybe add a linter script like
```
"lint-staged": {
    "linters": {
      "{src,scripts}/**/*.{js,jsx,json,scss,sass}": [
        "prettier --single-quote --write",
        "git add"
      ]
    }
  }
```

And then call it in the pre-commit
```
"husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  }
```

Comment (palashg7563) [reply server-api/package.json]: @abhishek71994 done!

**Comment by abhishek71994** [comment]
Looks good to me.

**Pull request merged**